### Sawyer Push ############################################
# desired final pose of the control joint
desiredFinalPos: [1.2, 0, 0.0647, 0.0, 0.0, 0.0]
# desired final velocity of the control and robot joints
desiredFinalVel: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
# initial index of the DOF associated with the control joint
controlJointDOF0: 0
# cost function weights
w1: 1e3		# weight on deviations in x and y directions
w2: 0e0		# weight on deviations in z and orientation
w3: 1e0		# weight on final velocities
w4: 2e-1	# weight on virtual stiffness
